sorc/build_ufs.sh

```diff
@@ -21,7 +21,7 @@ while getopts ":da:v" option; do
   esac
 done
 
-cd "${cwd}/ufs_model.fd"
+cd "${cwd}/ufs_model.fd" # TODO: This is redundant; see below.
```

Comment:
```suggestion
cd "${cwd}/ufs_model.fd"
```
No it is not!  If you delete this line, line 26 will fail.